test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -591,6 +591,14 @@ def func(tensor):
         tensor = common_utils.get_whitenoise(sample_rate=44100)
         self._assert_consistency(func, tensor)
 
+    def test_resample(self):
```

Comment:
Looks good. Given tests pass, can we parameterize `resample_method` to cover the case for `"kaiser_window"`?

Reply:
tests are failing locally after rebasing due to new changes with kaiser window and constructing a tensor out of an Optional[float] type, but I will look into it